src/app/event-external-root/event-external-root.service.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+angular.module('esn.calendar.event-consultation')
+  .factory('calEventExternalRootService', calEventExternalRootServiceFactory);
+
+function calEventExternalRootServiceFactory($http) {
+  return {
+    changeParticipation: changeParticipation,
+  };
```

Comment:
```js
  return {
    changeParticipation
  }
```

Reply:
you are right, I forgot that we have Babel now